### DISCUSSION TITLE
Implications of Idealism in Theoretical Physics

### DISCUSSION THREAD

**Discussion by khuramonline** [a-cosmology-group/acg#54] [Metaphysics and Philosophy]
Implications of idealism are that our own ideas seem more real than the actual physical facts. Rather than explaining unknown aspects of reality on the basis of known observed facts, science would start explaining or interpreting observed facts on the basis of metaphysical ideas of larger reality. Big Bang theory is one of extreme examples of this kind of manifestation of idealism in Theoretical Physics.

In this modern idealism, we start from GR equations. We find ‘correct solutions’ on paper **and take them for real**. It means we are at least at an advanced stage than those **ancient animist people for whom any idea would be as real as physical reality**. Our “modern science” has only sorted out that status of reality cannot be assigned to each and every idea of mind. This status goes only to ‘correct solutions’ of GR equations.

The actual physical reality is only that we are getting redshifted light from everywhere such that more distant galaxies are more redshifted. **But more real for scientists is the very first moment of creation of universe that is found out through so called ‘correct solutions and our scientists become able to tell us (like modern magicians) all the details of that early stage of universe as if it took place before their eyes.** Not only that, rather than theorizing larger reality on the basis of observed facts (scientific approach), they have started explaining or interpreting observed facts on the basis of so-called already known larger facts of reality.

Especially the Big Bang Cosmology tends to explain little (observable) facts on the basis of ‘already known’ realities of whole universe. For example, our scientist (Georges Lemaître) already knows that ‘Universe is Expanding’. On the basis of this ‘already known’ larger fact of reality, he ‘explains’ observed redshifts of galaxies (little fact).

Title of his (1927) paper is “A Homogeneous Universe of Constant Mass and Increasing Radius accounting for the Radial Velocity of Extra-galactic Nebulæ”.

Thus before explaining redshifts, he is already aware of ‘larger fact’ that radius of universe is increasing. On the basis of already known ‘larger fact’, he is explaining or interpreting a little (observable) fact.

CMB is also explained in this way. Science ‘already knows’ that universe originated from a point that, after 380000 years, had become 43 million light years in diameter. Science ‘already knows’ that this universe first time emitted light (photons). Science ‘already knows’ that that universe had such and such temperature. Science ‘already knows’ that universe was expanding and that light was also expanding. Science ‘already knows’ that now the original light must have this much temperature.

On the basis of these ‘already known’ larger realities of universe, CMB (a little observable fact) is explained.

‘Predictions’ of Big Bang Cosmology did not even match with observed fact. Therefore, adjustments of dark matter/dark energy are applied to get matching results. This is the actual prevailing scientific methodology whose examples are quite apparent in different aspects of the Big Bang Cosmology. And they say that it is science.

### Enforcement of Idealism Through Mathematics

There is one thing called **artificial intelligence**. Another thing is known as **artificial selection**. Theoretical Physics has adopted another thing that may be called **“artificial evolution of the scientific method”**. Using this method, Theoretical Physicists get sure knowledge that their understanding of physics is correct.

It is very easy in fact. Just work out such mathematics which keeps the final results of any experiment within certain limits (i.e. Enforcement of Circular Logic Through Mathematics). With this method, all the experiments will surely confirm their theory and physicists can confidently inform the world that their understanding of physics is correct.

### What is regarded as proof in this system?

In this system, proof is the real observation or experiment — as interpreted by their “mathematics” whose final result has to be within certain acceptable limits.

In this way, basically facts are first modified by applying such mathematics whose final result is pre-determined. Then obviously those modified results confirm the theories of this theoretical physics. And then a series of propaganda starts that their theories are confirmed by “every” experiment or observation.

Situation is that in Theoretical Physics, facts stay only in a modified form. For example fact of science is that there is linear redshift-distance relationship.

But “fact” of Theoretical Physics is that Universe is expanding at rate 67 Km/sec/mpc.

Another modified fact of Theoretical Physics is that there is no linear relationship of redshift and distance at higher redshifts. **This modified form is achieved when the actual fact of science is filtered through SR based redshift formulas.**

CMB is observation. Fact is only that it is a form of light coming from everywhere. But modified fact in Theoretical Physics is that it is like a picture of big bang.

Extreme example of modification of facts through formulas is that distances of galaxies are assigned through SR based formulas which overrule the fact of linear relationship of redshift-distance into non-linear relationship. **Through these formulas upper limit of distance is formed which is 13.6 billion light years.**

Now JWST or any other powerful telescope cannot see any galaxy beyond this distance **not because galaxies cannot be seen beyond that distance**. JWST or other powerful telescopes cannot see any galaxy beyond that distance because **distances of all the galaxies as not greater than 13.6 billion light years is pre-determined by the formulas**.

It is plain lie to say that Theoretical Physics is based on observations. No.

**Theoretical Physics is an extreme form of idealism where ideas (formulas) have decisive role.** Observations must obey those formulas. This system has nothing to do with reality. Even this system also has nothing to do with modeling of the reality. If observations do not tally with formulas then preferential treatment is to add dark stuff to the observations and then make that dark stuff part of your model. If formula says that there should be like entangled state then physical particles must be in entangled state and scientists are at full liberty to describe such fanciful experiments which they never conducted … like they observed and stored two entangled quantum particles … then they separated those entangled particles by many kilometers and “observed” properties were found exact in line with formulas. **Obviously only modified observations were found in line with their formulas.**

This system has nothing to do with constructing a model of reality. **At the most this system makes a model of a set of formulas and observations are modified so to obey that model.**

And then it is just repeatedly told that every observation confirms that “model”. Of course, every observation is first made conforming to the “model” **and then those modified observations do confirm that model.**

This post covers imprints of Idealism and Animism in modern Theoretical Physics. Core methodology of the Theoretical Physics is however based on Rationalism, details thereof warrant a separate post.

**Comment by OlliSant** [comment]
That is a very good explanation of the idealism of BB, what is wrong in that theory in the point of view of the scientific method, and what is its metaphysics. And what could be a better metaphysics. We cannot avoid metaphysical presuppositions. 

Briefly the same thing can be said so, that science tries to find, invent a theory that fits the observations in the best available way, and not try to fit the observations to existing, approved theory eternally, too long time, when the observations and rational thinking do not fit anymore with the old theory.  

The theories here are GR, BB and QT. BB must be abandoned, and better must be invented, GR needs revisions, and QT is maybe good enough in this connection where it is not so much needed. New theory for cosmology is needed, and the theory must be of the whole universe, not of the observable universe, which is smaller. 

When we have not a theory to replace BB, we can be eclectic, approve all the best theories in the same time, some steady state theories, some multiverse theories and some cyclic theories and some theories that unite them all. These three theories are the logical possibilities for universe, and we can make them also in the eternal and boundless base. The beginning is impossible, and the expansion is not needed when we change theory.

**Comment by khuramonline** [comment]
Here I want to add a little emphasis on one point of the main post. Following is the point:

> We find ‘correct solutions’ on paper and take them for real. It means we are at least at an advanced stage than those **ancient animist people for whom any idea would be as real as physical reality**. Our “modern science” has only sorted out that status of reality cannot be assigned to each and every idea of mind. This status goes only to ‘correct solutions’ of GR equations.

Animism is actually a raw form of drawing conclusions. This approach prevailed everywhere in anciant human tribes and this approach still springs in fertile cultures. Here "fertile culture" mean a culture where unrealistic attitudes or points of views are common.

Animism is also about the inability of mind to distinguish between fact and ideas. An animist person takes ideas for facts. Big Bang Cosmology has exposed the level of animistic aprroach within the science community. Here top level scientists can talk about details of the first three minutes after big bang as facts and they can also see the remnents of the previous universe within the JWST images.

Above two are the examples from top level scientists. Within **mid-tier scientific community**, following types of **"ideas"** are often treated or projected as **"facts"**.

1. Spacetime is a real thing. philosophers should know that space, time and spacetime all are ideas of mind.
2. "Early hot and dense universe" is a real thing. Realistic person should know that early hot and dense state is an idea that was derived from equations. There is no empirical evidence so far for a real hot and dense state.
3. Expansion of space is real.
4. Redshift means recessional velocity in real literal terms. 
5. CMB is really the exact same radiation that was predicted by the big bang cosmologists.
6. There was actually an inflation period within the first second of the birth of universe.
7. Big bang is an idea but mostly projected or treated as a fact. Top level science journals report distances of far off galaxies not like how much billion light years away but in terms of **how many hundred million years after the big bang**. Anything seen beyond that distance is understood as "remnents" of previous universe. 

In short -- within science literature there are lots of examples which indicate that science people are sufferring with a sort of inability to distingush between ideas and facts. **Technically, this inability is called Animism**.

[Locality History, Memory: The Making of the Citizen in South Asia](https://books.google.com.pk/books?id=JJwYBwAAQBAJ&pg=PR12&lpg=PR12&dq=khuram+rafique&source=bl&ots=LzJHJiqefk&sig=ACfU3U1ACeyycRLsZAAEtm1k2zAEaLCoEw&hl=en&sa=X&ved=2ahUKEwjclJbJte_lAhUE5uAKHfN-C384FBDoATAHegQICRAB#v=onepage&q=khuram%20rafique&f=false). Cambridge University Press (2009)

Above is link to google books preview of a book. I am a chapter author in this book. Fortunately, almost complete Chapter 3 written by me is available in this google books preview.

Title of Chapter is: **"Animism and Mythology--Reflections of Human Essence."**

**Reply by OlliSant** [reply]
Good, but the spacetime is also in reality, it is a model of reality, right or wrong, it is not only in the mind.

**Reply by khuramonline** [reply]
Yes you are right. Spacetime does have "non real object" explanation or even definition in term of contnuum. I am ready to exclude this point.

**Comment by budrap00** [comment]
Space, time and spacetime are relational concepts in the same way that
distance and temperature are relational concepts. No one thinks distance
"causes" you to be separated from the wall and no one thinks something
called temperature causes things to get hotter or colder - they are
relational concepts.

Space and time are exactly the same. They are useful concepts as long as
you don't reify them into physical entities that causally interact with
matter and energy. There is absolutely no empirical evidence for the
existence of a causally interacting space, time, or spacetime.
Bud

**Comment by khuramonline** [comment]


**Reply by RedshiftDrift** [reply]
This comment contravenes the ACG guidelines set out in [*Web Content and Discussion Guidelines*](https://github.com/a-cosmology-group/acg/blob/main/CONTRIBUTING.md).

Specifically, no first degree relationship is shown between quantum mechanics' *spooky action at a distance* and cosmology.

You may wish to keep a copy, it will be removed tomorrow.

**Reply by khuramonline** [reply]
Sir, Give me a chance to explain that topic was not even about QM and QM's spooky action at a distance.

Official argument is that Newton's gravity theory failed to overcome the spooky action at a distance. That was not spooky action at quantum scale. And Newton had acknowledged this problem.

Second official argument is that Einstein's spacetime curvature has solved this spooky action at a distance problem. Again it is not about QM's spooky action.

My argument is only denying the second official argument. I am only saying that spooky action at distance problem is not solved at non-quantum scale within the context of gravity by using the idea or concept of spacetime.

Now how does it relate with Cosmology?

Since whole modern Cosmology has been built around GR ... by using the name of GR. So true nature of GR must be sorted out.

In fact at the base it is gravity itself. 

When official Cosmology has come from the theory of gravity, then Gravity, and all aspects of gravity do have first degree relationship with Cosmology.

**Reply by RedshiftDrift** [reply]
Your reasoning
> Since whole modern Cosmology has been built around GR ... by using the name of GR. So true nature of GR must be sorted out.

is the fallacy of ["Affirming the Consequent"](https://en.wikipedia.org/wiki/Affirming_the_consequent).  You take a true [conditional](https://en.wikipedia.org/wiki/Indicative_conditional) statement ("if GR were broken, then modern Cosmology would be wrong"), and invalidly infer its converse ("modern Cosmology is wrong, so GR must be broken").

Moreover, an alternative to GR is speculative, so it relates with cosmology to the second degree at best.

**Reply by khuramonline** [reply]
Sir, It is not like that. Following are my words:

> Modern Cosmology has been built around GR ... **by using the name of GR**.

Here I used the phrase "by using the name of GR".

Because my considered stance is that GR has only superficial relationship with expanding universe theory. Expanding Universe Theory even have fraudulent nexus with GR. Following is from my first book (2018):

![image](https://github.com/a-cosmology-group/acg/assets/132253587/ee995c05-f778-4b82-9c8c-c8d54a10faea)

I have written similar things in other parts of book as well. 

**I do not say directly or indirectly that if official cosmology is wrong then GR also has to be wrong.**

However, evaluating every aspect of GR does have first degree relation with Cosmology. Because true nature of GR itself will break the official Cosmology.

What I think is that GR is important and it is an achievement. But it is not absolute achievement. Neither it has solved Newton Theory's outstanding problem of spooky action at a distance. If Newton's gravity theory was 67% achievement then GR is also 65% achievement.

**Comment by OlliSant** [comment]
The problem is that the spacetime in GR is not final and right. In cosmology the matter is galaxies and forces. Space is immensely big, time is for particles in space. For the universe as a whole, space is 4D, time is not a dimension, is not beginning, is eternal. Gravity is a real force, and there is more electromagnetic forces as supposed now.

From the astronomical findings and classical philosophy I can already say, which kind of universe the universe is. 

**The universe  is galaxies, the web of galaxies everywhere and always.** Locally there are crunches, explosions and star and galaxy formation also eternally and everywhere. 

This is published in my article in ACG Essays and models in 2022.

Only the dogmatic BB theory hinders people to see this simple fact, found now by astronomy.

**Comment by OlliSant** [comment]
I mean that from the observations and all real knowledge in astronomy and thoughts in philosophy, every philosopher can now tell us, which kind of universe we have in the reality. This really is so simple. We can see it from science, philosophy and modern observations. We can reason it like Sherlock Holmes. There are enough facts and clues. Just reason the rest.

Just forget the BB, expanding and beginning universe. It is wrong, so simple.
